src/org/mozilla/javascript/Parser.java

```diff
@@ -3160,16 +3171,20 @@ private AstNode parenExpr() throws IOException {
             Comment jsdocNode = getAndResetJsDoc();
             int lineno = ts.lineno;
             int begin = ts.tokenBeg;
-            AstNode e = (peekToken() == Token.RP ? new EmptyExpression(begin) : expr());
+            AstNode e = (peekToken() == Token.RP ? new EmptyExpression(begin) : expr(true));
```

Comment:
To my points above, if this is the only place where we pass "true," does it make more sense to add a new function instead?